spectrocloud/resource_cluster_aws.go

```diff
@@ -58,6 +58,14 @@ func resourceClusterAws() *schema.Resource {
 				},
 				Description: "A list of tags to be applied to the cluster. Tags must be in the form of `key:value`.",
 			},
+			"tags_map": {
```

Comment:
@stefanSpectro we need to restrict the user to stick with either one attribute `tags` or `tags_map`